opsdroid/connector/matrix/tests/responses/single_message_sync.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Note for future self:

I should write a sync response factory. If I clean out this json file so it's a template but with no data, a factory could generate a response with one or more events injected into it at the right point. This would prevent us from having to save a load of custom sync responses for each event we want to test.